setup.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+from setuptools import setup, find_packages
+
+__version__ = "0.0.1"
```

Comment:
If we define version number inside setup.py, we won't be able to [check this package's version on-the-fly](https://www.python.org/dev/peps/pep-0396/#examples). Would you consider [do this](https://stackoverflow.com/a/39671214/728675) insead? <sub>Shameless advertising: I wrote that SO answer.</sub>

Reply:
Fixed in 8305767

Reply:
Thanks for showing me this! It definitely demonstrates my general noobiness in Python.

Reply:
It is not about noobiness at all! Organizing a python package is a very specific area that most Python developers - who are library consumers - would never touch, because they don't need to.